src/lib/components/LinksMenu.svelte

```diff
@@ -6,7 +6,7 @@
 
   export let ref: Element | undefined = undefined;
 
-  export let className: string;
+  export let style: 'primary'|'secondary'|'ternary';
```

Comment:
I think the technically correct term for "third one" is "tertiary" instead of "ternary", but I'm okay if you want to leave it.

Reply:
I'll change that. Not sure why "ternary" popped in my head. thanks!